app/controllers/concerns/remember_device_concern.rb

```diff
@@ -77,6 +77,6 @@ def handle_valid_remember_device_analytics
   end
 
   def remember_device_cookie_expiration
-    IdentityConfig.store.remember_device_expiration_hours_aal_1.hours.from_now
+    nil
   end
```

Comment:
We should feature flag this so we can test out the user-facing impact of it, since browsers have different interpretations of what "session" expiration is

Reply:
done!